spec/controllers/idv/by_mail/enter_code_controller_spec.rb

```diff
@@ -34,6 +34,8 @@
         :gpo_confirmation_code,
         profile: pending_profile,
         otp_fingerprint: Pii::Fingerprinter.fingerprint(otp),
+        created_at: 2.days.ago,
+        updated_at: 2.days.ago,
       )
```

Comment:
It might be time to add a GpoConfirmationCode factory.

Reply:
There is one and I'm using it here. Are you suggesting adding a trait? I thought about it, and if I'd needed to do this more than once, I would.